device/rsvp/rsvpTspec.yaml

```diff
@@ -0,0 +1,42 @@
+components:
+  schemas:
+    Rsvp.Tspec:
+      description: >-
+        Configuration for RSVP-TE TSPEC object included in Path Messages. The usage of these parameters is defined in RFC2215.
+      type: object
+      properties:
+        token_bucket_rate:
+          description: >-
+            The rate of the traffic to be carried in this LSP in bytes per second.
+            This is part of the Token Bucket specification defined for a traffic flow defined in RFC2215.
+          type: string
+          format: float
+          default: '0'         
+        token_bucket_size:
+          description: >-
+            The depth of the token bucket in bytes used to specify the Token Bucket characteristics of the traffic to be carried
+            in the LSP. This is part of the Token Bucket specification defined for a traffic flow defined in RFC2215.
```

Comment:
The depth of the token bucket in bytes for a  LSP. This is part of the Token Bucket specification defined for a traffic flow in RFC2215.

Reply:
Done.

Reply:
Done.